packages/internal/dex/sdk-sample-app/src/components/Example.tsx

```diff
@@ -35,10 +37,10 @@ const allTokens: Token[] = [
   { symbol: 'zkYEET', address: '0x8AC26EfCbf5D700b37A27aA00E6934e6904e7B8e' },
 ];
 
-const buildExchange = (
-  secondaryFeeRecipient: string,
-  secondaryFeePercentage: number
-) => {
+/// @dev Update this to change tokens used in the app
+const allTokens = testnetTokens;
+
```

Comment:
How is this being used?

Reply:
It's just to make flipping between the token lists a bit easier. Ideal solution is to have a dropdown that allows selecting the chain, but that's out of scope for this I think